caasp-389-ds-image/root/usr/local/bin/entrypoint.sh

```diff
@@ -27,38 +27,69 @@ if [ ! -e "/data/config/setup.inf" ]; then
 
     echo -e "\n>> Creating localhost configuration...\n"
     cat <<EOT >> /data/config/setup.inf
-[General]
-FullMachineName=localhost.${DOMAIN_SUFFIX}
-;SuiteSpotUserID=dirsrv
-;SuiteSpotGroup=dirsrv
-;Should not run as root
-SuiteSpotUserID=root
-;Should not run as root
-SuiteSpotGroup=root
-StrictHostCheck=false
+[general]
+config_version = 2
+;defaults = 999999999
+;full_machine_name = localhost.${DOMAIN_SUFFIX}
+selinux = False
+start = False
+strict_host_checking = False
+systemd = False
+start_server = False
+with_systemd = False
 
 [slapd]
-ServerPort=3389
-;SecurePort=3636
-ServerIdentifier=localhost
-Suffix=${DS_SUFFIX}
-RootDN=${ROOT_DN}
-RootDNPwd=${DS_DM_PASSWORD}
-ds_bename=userRoot
-SlapdConfigForMC=Yes
-UseExistingMC=0
-AddSampleEntries=No
-start_server=0
-with_systemd=0
-;self_sign_cert=True
-
+backup_dir = /data/bak
+;bin_dir = /usr/bin
+;cert_dir = /etc/dirsrv/slapd-localhost
+;config_dir = /etc/dirsrv/slapd-localhost
+;data_dir = /usr/share
```

Comment:
Should we remove these commented out settings?

Reply:
I create this from CLI 'dscreate create-template'. I keep these variables, so I know what is default value. It can be removed.